stacks/ActivitiesStack.tsx

```diff
@@ -153,7 +160,11 @@ export function TabNavigation () {
         initialParams={{ event: events[2] }}
       />
       <Tab.Screen name="Notifications" component={TestNotifScreen} />
-      <Tab.Screen name="Profile" component={TestProfileScreen} />
+      <Tab.Screen
+        name="Profile"
+        component={ProfileScreenNavWrapper}
+        initialParams={{user: UserMocks.Mia}}
```

Comment:
Better to feed this data from the "navWrapper" than through initialParams, so later it'll be easier to swap out with the live query